inlong-sort/sort-flink/sort-flink-v1.13/sort-connectors/hive/src/main/java/org/apache/inlong/sort/hive/HiveTableSink.java

```diff
@@ -170,23 +188,50 @@ private DataStreamSink<?> consume(
         try (HiveMetastoreClientWrapper client =
                 HiveMetastoreClientFactory.create(HiveConfUtils.create(jobConf), hiveVersion)) {
 
-            Table table = client.getTable(dbName, identifier.getObjectName());
-            StorageDescriptor sd = table.getSd();
-
-            Class hiveOutputFormatClz =
-                    hiveShim.getHiveOutputFormatClass(Class.forName(sd.getOutputFormat()));
+            StorageDescriptor sd;
+            Properties tableProps = new Properties();
+            Class hiveOutputFormatClz;
+            HiveWriterFactory writerFactory;
+            boolean sinkMultipleEnable = Boolean.parseBoolean(
+                    catalogTable.getOptions().getOrDefault(SINK_MULTIPLE_ENABLE.key(), "false"));
             boolean isCompressed =
                     jobConf.getBoolean(HiveConf.ConfVars.COMPRESSRESULT.varname, false);
-            HiveWriterFactory writerFactory =
-                    new HiveWriterFactory(
-                            jobConf,
-                            hiveOutputFormatClz,
-                            sd.getSerdeInfo(),
-                            tableSchema,
-                            getPartitionKeyArray(),
-                            HiveReflectionUtils.getTableMetadata(hiveShim, table),
-                            hiveShim,
-                            isCompressed);
+            if (sinkMultipleEnable) {
+                sd = new StorageDescriptor();
+                SerDeInfo serDeInfo = new SerDeInfo();
+                serDeInfo.setSerializationLib("org.apache.hadoop.hive.serde2.lazy.LazySimpleSerDe");
```

Comment:
Maybe, this param can be passed by HiveOptions, deafult valuse is "org.apache.hadoop.hive.serde2.lazy.LazySimpleSerDe";